README.md

```diff
@@ -65,10 +65,6 @@ To get more help, support and information please open a github [issue](https://g
 Contributing information can be found in the [CONTRIBUTING.md](CONTRIBUTING.md) file.
 
 ## TODO
-* implement ability for yeoman-ui to call methods in vscode extensions
-    * support hook for executing commands after finish
-    * support open readme.md file by default (with turn off setting)
-
 * support extensible question type custom user interfaces for complex operations (e.g. choose odata source)
```

Comment:
I believe these 2 are already supported. We should probably delete the TODO section, as we now maintain feature requests as GitHub issues.